index.html

```diff
@@ -0,0 +1,136 @@
+    </div>
+    <div class="program-item pt-2 mx-1 mt-3 px-3">
+      <div class="program-logo"><i class="fa-regular fa-gift"></i></div>
+      <div class="pro-topic"><h3>Award Presentation</h3></div>
+      <div class="prosub-topic text-light text-center"><p>Participants will get a chance to present their portfolio to sponsors. The winner with the best portfolio will also get $1000 cash price and instant employment</p></div>
+    </div>
+    <div class="program-item pt-2 mx-1 mt-3 px-3">
+      <div class="program-logo"><i class="fa-regular fa-people-line"></i></div>
+      <div class="pro-topic"><h3>Interactive Lecture</h3></div>
+      <div class="prosub-topic text-light text-center"><p>Interactive session with leading women in Tech. The Lecture will be conducted in large groups.</p></div>
+    </div>
+    <div class="program-item pt-2 mt-3 px-3">
+      <div class="program-logo"><i class="fa-solid fa-handshake"></i></div>
+      <div class="pro-topic"><h3>Networking</h3></div>
+      <div class="prosub-topic text-light text-center"><p>Participants will get a chance to connect and network with other women in different tech communities during this event. This will be a great opportunity build relations.</p></div>
+    </div>
+  </div>
+    <div class="program-bo d-flex flex-column justify-content-center align-items-center">
+      <h2 class="h4 text-center prog-see py-5"><a href="#" class="text-dark">See more Programs</a></h2>
+    </div>
+    <div class="join container align-items-center justify-content-center px-3"><a href="#" class="joina"> <h3 class="text-center text-white">Join Women In Tech Summit</h3></a> </div>
+</section>
+<!-- FEATURED SPEAKERS -->
+<Section class="featured-speakers container py-2">
+  <div class="program-box d-flex flex-column justify-content-center align-items-center">
+    <h2 class="h2 text-center pt-5">Featured Speakers</h2>
+  <div class=" underprogram mb-4"></div>
+  <div class="featured">
+  </div>
+</Section>
+
+<!-- PARTNERS -->
+<section class="partners container-fluid">
+  <div class="partner-main py-3 px-2">
+    <div class="program-box d-flex flex-column justify-content-center align-items-center">
+      <h2 class="h2 text-center pt-5 text-white-50">Partner</h2>
+    <div class=" underprogram mb-4"></div>
+  </div>
+  <div class="partner-cont container flex-row d-md-flex">
+    <div class="partner-img mx-2 my-2"><a href="https://www.google.com/"><img src="./images/google.avif" alt="" class="image-part"></a></div>
+    <div class="partner-img mx-2 my-2"><a href="https://www.microsoft.com/"><img src="./images/microsoft.png" alt="" class="image-part"></a></div>
+    <div class="partner-img mx-2 my-2"><a href="https://www.boomeranghq.net/"><img src="./images/boomerang.jpg" alt="" class="image-part"></a></div>
+    <div class="partner-img mx-2 my-2"><a href="https://www.amazon.com/"><img src="./images/amazon.jpg" alt="" class="image-part"></a></div>
```

Comment:
- [Optional] Consider adding a value to the `alt` attribute in your `img` tags that briefly describes the image. This is so that screenreaders can know what image it is and also so that the text can be displayed when the image fails to load.